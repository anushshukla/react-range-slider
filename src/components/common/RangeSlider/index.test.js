import React from 'react';
import renderer from 'react-test-renderer';
import WithDualRangeSliderHOC from 'components/common/RangeSlider';
// import ReactTestUtils from 'react-dom/test-utils';

const DummyComponent = () => <span />;

const ComponentToTest = WithDualRangeSliderHOC(DummyComponent);

const props = {
  from: 21,
  to: 71
};

describe('context ', () => {
  const component = renderer.create(<ComponentToTest {...props} />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
