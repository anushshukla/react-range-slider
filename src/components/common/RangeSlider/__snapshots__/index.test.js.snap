// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`context  should match snapshot 1`] = `<span />`;
